test/md040-extra.md

```diff
@@ -0,0 +1,25 @@
+# md040-extra.md
+
+Fence code block information with leading whitespaces:
+
+```  html
+<h1>markdownlint</h1>  {MD040:5}
+```
+
+Fence code block information with trailing whitespace:
+
+```css 
+body {}  {MD040:11} {MD009:11}
```

Comment:
It's not unreasonable, but should we really warn for both rules in this scenario? Per the specification, trailing white space will already be removed and so is harmless and arguably not "extra" data. I'm thinking maybe only the trailing white space rule should report for this example. What do you think?